### PR TITLE
adjusted the small text edits

### PR DESCRIPTION
**Pull request by SanliFaez**
issue #2 can be closed
skipping the change of slug from answer -> vragen. there are some internal links that have used the previous slug and this might break those.

### CODE REVIEW

File: klimaat_helpdesk/templates/core/includes/footer_text.html [hunk 1]
```diff
@@ -9,7 +9,7 @@
 
     <div class="footer-bar__links">
       <span class="footer-bar__title">Een initiatief van
-        <a class="footer-bar__title-link" href=https://scientists4future.nl/">scientists for future</a> &
+        <a class="footer-bar__title-link" href=https://scientists4future.nl/">Scientists4Future NL</a> &
```
Comment (aquilesC) [inline klimaat_helpdesk/templates/core/includes/footer_text.html:12]: This link is badly formatted (missing opening ")

File: klimaat_helpdesk/templates/core/question_submitted.html [hunk 2]
```diff
@@ -45,7 +45,7 @@ <h1 class="form__header">{% trans 'Bedankt voor je vraag!' %}</h1>
             {{ form.accept_terms.label_tag }}
             {{ form.accept_terms }}
             <p>
-              <a href="/terms" target="_blank">{% trans "Our terms and conditions" %}</a>
+              <a href="/terms" target="_blank">{% trans 'Onze voorwaarden' %}</a>
```
Comment (aquilesC) [inline klimaat_helpdesk/templates/core/question_submitted.html:48]: Should it be /terms or /voorwaarden ?

File: klimaat_helpdesk/templates/core/includes/footer_text.html (hunk 1 above)
Comment (SanliFaez) [reply klimaat_helpdesk/templates/core/includes/footer_text.html]: fixed in the new pull request

File: klimaat_helpdesk/templates/core/question_submitted.html (hunk 2 above)
Comment (SanliFaez) [reply klimaat_helpdesk/templates/core/question_submitted.html:48]: i am still using /terms. I don't get the obsession with slugs been dutch or english. makes translation of the website only more difficult!
so no changes here

**Comment by SanliFaez** [comment]
I also added the google analytics tag into base.html in the new pull request. I hope it has come in correctly. 

the .gitignore is also killing me because it does not behave {angry face}

**Pull request merged**